src/appleseed-max-impl/appleseeddisneymtl/appleseeddisneymtl.cpp

```diff
@@ -792,7 +792,9 @@ asf::auto_release_ptr<asr::Material> AppleseedDisneyMtl::create_max_material(
     asr::Assembly&  assembly,
     const char*     name)
 {
-     asr::ParamArray material_params;
+    asr::ParamArray material_params;
+    std::string texture_name;
+    bool use_max_procedural_maps = true;
```

Comment:
Hi @dictoon, this variable is used in place of bool value `true` in all `insert_texture_and_instance()` calls. Is it ok or i should be using `true` instead?